run.py

```diff
@@ -684,6 +684,8 @@ def fetch_test_details(var) -> dict:
                 config["base_url"] = base_url
 
             config["rhbuild"] = rhbuild
+            config["version2"] = version2
```

Comment:
`version2` is a temporary change and would go away in some time. Hence, I would like to avoid using it... instead, we could do this to support backward compatability.

```
config["rhbuild"] = f"{rhbuild}-{platform}" if version2 else rhbuild
if version2:
  config["platform"] = platform